README.md

```diff
@@ -166,7 +166,7 @@ Expresses the portion of Surplus Weight which has been refunded. Not used on XCM
 
 Of type `MaybeErrorCode`, initialized to `MaybeErrorCode::Success`.
 
-Expresses the result of a `Transact` instruction after the encoded call within has been dispatched.
+Expresses the result of a [`Transact`](https://github.com/paritytech/polkadot/blob/962bc21352f5f80a580db5a28d05154ede4a9f86/xcm/src/v3/mod.rs#L482) instruction after the encoded call within has been dispatched.
```

Comment:
Feels like this format spec should be as detached as possible from the polkadot implementation, for modularity's sake. I still think it's useful to point to the main implementation, maybe on its own separate section in the readme? Something like `Main implementation(s)`. What do you think?

Reply:
For now, I feel it is fine, as there is only a main implementation currently reference-able (to my knowledge).